MAINTAINERS

```diff
@@ -23,11 +23,10 @@
 		]
 
 	[Org."Docs maintainers"]
-
-	# TODO Describe the docs maintainers role.
-
 		people = [
-			"moxiegirl"
+			"sfsmithcha",
+			"sven",
```

Comment:
This is the "handle" we use for @SvenDowideit in other repositories, so I sticked with that.

w.r.t. the GitHub ID; his GitHub handle is with uppercase `S` and `D`, so we use that as well (although on GitHub, it is case-insensitive - not sure if the GitHub API is case-insensitive as well)